install.sh

```diff
@@ -17,9 +17,9 @@ function yellow {
     printf "${YELLOW}$@${NC}\n"
 }
 
-curl -s https://raw.githubusercontent.com/jschmid1/gopro_as_webcam_on_linux/master/gopro -o /usr/local/sbin/gopro
+cp gopro /usr/local/sbin/
```

Comment:
Where do you get this from?

Reply:
Oh, actually, that change can be reverted. Let me fix that.

Reply:
All done!